bin/create-component

```diff
@@ -15,3 +15,4 @@ for i in ".spec." ".story." "."; do mv $component_path/{Component,$component_nam
 find ${component_path} -type f -iname "$component_name*js" -exec sed -i.bak "s/Component/$component_name/g" {} \;
 find ${component_path} -type f -iname "index.js" -exec sed -i.bak "s/Component/$component_name/g" {} \;
 rm ${component_path}/*.bak
+echo "export { default as $component_name } from './components/$component_name' };" >> ./src/index.js
```

Comment:
Hmmmm, I think this needs a bit more logic. When someone is adding a sub-component via the second argument of the script, you don't want to add the export. So I guess checking for a truthy `$component_sub_path` before doing this would make sense.

Reply:
Ah, i didn't realize this functionality existed. I can amend this.